fluentd/vendored_gem_src/syslog_protocol/lib/syslog_protocol/packet.rb

```diff
@@ -68,7 +68,8 @@ def tag=(t)
       unless t && t.is_a?(String) && t.length > 0
         raise ArgumentError, "Tag must not be omitted"
       end
-      if t.length > 32
+      #if t.length > 32
+      if t.length > 500
```

Comment:
Does it make sense to add replace this with a constant someplace which is more descriptive?